tools/docker/finish_bazel_install.sh

```diff
@@ -0,0 +1,12 @@
+set -e
+# hack to have the dependecies in the docker cache
+# and have much faster local build with docker
+# can be removed once docker buildx/buildkit is stable
+# and we'll use "RUN --mount=cache ..." instead
```

Comment:
Just curious, any information about when this will be removed remove experimental? Docker 20?

Reply:
No idea. This project is public since quite some time now 2 years if I remember well. I hope it gets released soon. It has many other goodies that I'm super interested in using.